Lib/test/test_exceptions.py

```
@@ -561,6 +561,9 @@ def test_notes(self):
                 e.__notes__ = 42
                 self.assertEqual(e.__notes__, 42)
 
+                e.add_note("will not work")
```

Hm, I'd expect it to fail in this case, not silently swallow the note.